yt/visualization/volume_rendering/scene.py

```diff
@@ -359,12 +365,12 @@ def save(self, fname=None, sigma_clip=None, render=True):
 
     def save_annotated(
         self,
-        fname=None,
-        label_fmt=None,
-        text_annotate=None,
-        dpi=100,
-        sigma_clip=None,
-        render=True,
+        fname: Optional[str] = None,
+        label_fmt: Optional[str] = None,
+        text_annotate: Optional[str] = None,
+        dpi: int = 100,
+        sigma_clip: Optional[float] = None,
```

Comment:
This looks like it's causing a failure on jenkins.

Reply:
in fact the issue was that I wrote `Optional[int, float]`, which is a syntax error, instead of `Optional[Union[int, float]]`. It is solved now !